src/Monsters/Races/Monsters/Dragons.js

```diff
@@ -0,0 +1,311 @@
+                        <td>4-5</td>
+                        <td>Dungeon Room Treasure and Gold</td>
+                        <td>
+                            Make one roll on the Dungeon Room Treasure Table for each Warrior to determine what he
+                            finds.<br />
+                            In addition, each player may roll as many D6 as he likes, adding up the score and
+                            multiplying it by 10 to determine how much extra gold his Warrior finds. However, if any of
+                            the dice score a 1 his Warrior finds no extra gold.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>6+</td>
+                        <td>Objective Room Treasure</td>
+                        <td>
+                            Make one roll on the Objective Room Treasure Table for each Warrior to determine what he
+                            finds.
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        `,
+    };
+}
+
+export function Dragon() {
+    const dragonType = DragonType();
+    return {
+        singularName: `${dragonType.name} Dragon`,
+        pluralName: `${dragonType.name} Dragons`,
+        description,
+        race: Monsters,
+        wounds: 74,
+        move: 6,
+        weaponSkill: 6,
+        ballisticSkill: undefined,
+        strength: 6,
+        toughness: 6,
+        initiative: 8,
+        attacks: 7,
+        gold: 4500,
+        armour: 6,
+        damage: "6D6",
+        specialRules: [
+            AmbushMagicModifier(dragonType.specialRule, "A"),
+            Fly(),
+            IgnoreBlows(5),
+            IgnorePain(7),
+            LargeMonster(),
+            Terror(12),
+            treasureHorde(0),
+        ],
+    };
+}
+
+export function GreatDragon() {
+    const dragonType = DragonType();
+    return {
+        singularName: `${dragonType.name} Great Dragon`,
+        pluralName: `${dragonType.name} Great Dragons`,
+        description,
+        race: Monsters,
+        wounds: 84,
+        move: 6,
+        weaponSkill: 7,
+        ballisticSkill: undefined,
+        strength: 7,
+        toughness: 7,
+        initiative: 7,
+        attacks: 8,
+        gold: 6000,
+        armour: 7,
+        damage: "6D6 / 7D6 (5+)",
+        specialRules: [
+            AmbushMagicModifier(dragonType.specialRule, "A"),
+            Fly(),
+            IgnoreBlows(4),
+            IgnorePain(7),
+            LargeMonster(),
+            Terror(13),
+            treasureHorde(1),
+        ],
+    };
+}
+
+export function EmperorDragon() {
+    const dragonType = DragonType();
+    return {
+        singularName: `${dragonType.name} Emperor Dragon`,
+        pluralName: `${dragonType.name} Emperor Dragons`,
+        description,
+        race: Monsters,
+        wounds: 94,
+        move: 6,
+        weaponSkill: 8,
+        ballisticSkill: undefined,
```

Comment:
Would it make more sense for this to be null instead of undefined, just so the code is consistent?